src/main/scala/streaming/SampleStatusesHashtagPrinter.scala

```diff
@@ -8,23 +8,19 @@ object SampleStatusesHashtagPrinter extends App {
 
   val client = TwitterStreamingClient()
```

Comment:
I would add the same TODO comment for consistency:
`// TODO - Make sure to define your consumer and access tokens!`